webpack/configs/base.ts

```diff
@@ -0,0 +1,36 @@
+import * as webpack from "webpack";
+
+import { TsConfigPathsPlugin } from "awesome-typescript-loader";
+
+const CommongConfig: webpack.Configuration = {
+
+    entry: "./webpack/globalize.ts",
+
+    output: {
+        path: `${__dirname}/../../build`
+    },
+
+    resolve: {
+        extensions: [".ts", ".mjs", ".js", ".json"],
+        plugins: [new TsConfigPathsPlugin()]
+    },
+
+    module: {
+        rules: [
+            // All files with a '.ts' extension will be handled by 'awesome-typescript-loader'.
+            { test: /\.ts$/, loader: "awesome-typescript-loader" },
+
+            { test: /\.mjs$/, include: /node_modules/, type: "javascript/auto" },
```

Comment:
Add a comment to this line explaining that this is for (like the lines above and below.

Reply:
Thanks for pointing this out to me - we actually don't need this for this project, so i'm removing this as well

---
In reply to: [250238437](https://github.com/Microsoft/clarity-js/pull/126#discussion_r250238437) [](ancestors = 250238437)